tools/docker/builder/openwrt/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update \
     && apt-get -y install --no-install-recommends \
     build-essential libncurses5 libncurses5-dev python python3 \
     unzip git ca-certificates gawk wget file bash \
-    python-yaml rsync less vim gnupg software-properties-common \
+    python-yaml python3-yaml rsync less vim gnupg software-properties-common \
```

Comment:
(nitpick) Commit message mentions that yq is added here, but actually it was already added in a previous commit. However, python3-yaml is added here without explanation. It is surprising that we would need both python-yaml and python3-yaml, can you explain that?

Reply:
python-yaml is needed by prplMesh, python3-yaml is needed by gen_config.yml (it's python3.6+ only).

Reply:
(I updated the commit message to reflect that)